cmdexec/cmdexec.go

```diff
@@ -21,13 +22,18 @@ import (
 // to stdout.
 // Once execution finishes, previous CWD is restored,
 // and the command output is returned as a string
-func ExecCommand(cmdSlice []string, stdinBody string, apiDir string) (httpie.ExResponse, error) {
+func ExecCommand(cmdSlice []string, stdinBody string, apiDir string) (httpie.ExResponse, int64,  error) {
 	oldDir, _ := os.Getwd()
 	utils.ChangeWorkingDir(apiDir)
+	start := time.Now()
```

Comment:
We are checking time http request only ?

Reply:
yes  the time difference between executing the API request